src/libexpr/fetchurl.nix

```diff
@@ -12,13 +12,13 @@
 , executable ? false
 , unpack ? false
 , name ? baseNameOf (toString url)
+, __impure ? false
```

Comment:
```suggestion
, impure ? false
```

The `__` prefix is appropriate for builtins like `derivation` to prevent them from clashing with user attributes, but that's not a danger here.